internal/core/model.rs

```diff
@@ -1204,7 +1208,7 @@ fn test_vecmodel_set_vec() {
     }
 
     let view = Rc::pin(TestView::default());
-
+    //iaosjdfk
```

Comment:
!!!

Reply:
ahh. I just fished this out of the stash. should have checked it better.